api/config/v1alpha1/metric_types.go

```diff
@@ -0,0 +1,60 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+type ProxyMetric struct {
+	// Prometheus defines the configuration for Admin endpoint `/stats/prometheus`.
+	Prometheus *PrometheusProvider `json:"prometheus,omitempty"`
+	// Sinks defines the metric sinks where metrics are sent to.
+	Sinks []MetricSink `json:"sinks,omitempty"`
+	// HistogramBucketSettings defines rules for setting the histogram buckets.
+	// Default buckets are used if not set. See more details at https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/stats.proto.html#config-metrics-v3-histogrambucketsettings.
+	HistogramBucketSettings []HistogramBucketSetting `json:"histogramBucketSettings,omitempty"`
+}
+
+type MetricSinkType string
+
+const (
+	MetricSinkTypeOpenTelemetry MetricSinkType = "OpenTelemetry"
+)
+
+type MetricSink struct {
+	// Type defines the metric sink type.
+	// EG currently only supports OpenTelemetry.
+	// +kubebuilder:validation:Enum=OpenTelemetry
+	// +kubebuilder:default=OpenTelemetry
+	Type MetricSinkType `json:"type"`
+	// OpenTelemetry defines the configuration for OpenTelemetry sink.
+	// It's required if the sink type is OpenTelemetry.
+	OpenTelemetry *OpenTelemetrySink `json:"openTelemetry,omitempty"`
+}
+
+type OpenTelemetrySink struct {
+	// Backend defines the backend to send OpenTelemetry metrics to.
+	// +kubebuilder:default={port: 3417}
+	Backend BackendService `json:"backend"`
```

Comment:
other Telemetry Sink APIs dont seem to have the `Backend` field
https://github.com/envoyproxy/gateway/blob/609949d6dfcc3726a78a7d6cbb7b6298883159fa/api/config/v1alpha1/accesslogging_types.go#L89
https://github.com/envoyproxy/gateway/blob/609949d6dfcc3726a78a7d6cbb7b6298883159fa/api/config/v1alpha1/tracing_types.go#L38
would be good to add the same design philosophy to all of them

Reply:
ok, let's keep same first, and change all of them later.